### DISCUSSION TITLE
training m values for same column used in multiple comparisons

### DISCUSSION THREAD

**Discussion by kotoroshinoto** [moj-analytical-services/splink#671] [General]
```

"comparisons": [
        jaccard_at_thresholds("NAME_CLEANED",include_exact_match_level=False),
        jaccard_at_thresholds("UNIVERSITY__C_CLEANED",include_exact_match_level=False),
        jaccard_at_thresholds("BILLINGSTREET", term_frequency_adjustments=True,include_exact_match_level=False),
        levenshtein_at_thresholds("NAME_CLEANED", distance_threshold_or_thresholds=[2,5],include_exact_match_level=False),
        levenshtein_at_thresholds("UNIVERSITY__C_CLEANED", distance_threshold_or_thresholds=[2,5],include_exact_match_level=False),
        levenshtein_at_thresholds("BILLINGSTREET", term_frequency_adjustments=True, distance_threshold_or_thresholds=[2,5],include_exact_match_level=False),
        exact_match("NAME_CLEANED"),
        exact_match("UNIVERSITY__C_CLEANED"),
        exact_match("BILLINGSTREET"),
        exact_match("EMAIL__C"),
]
```
when training I get this message:

Your model is not yet fully trained. Missing estimates for:
    - NAME_CLEANED (no m values are trained).
    - UNIVERSITY__C_CLEANED (no m values are trained).
    - BILLINGSTREET (no m values are trained).
    - NAME_CLEANED (no m values are trained).
    - UNIVERSITY__C_CLEANED (no m values are trained).
    - BILLINGSTREET (no m values are trained).
    
    I note that it only complains twice, so the first set (in order, it would be the jaccard at threshholes comparisons) seem to be getting trained just fine.
    
    Is the logic simply unable to handle this kind of setup? Do i have to combine them all into a single comparison somehow?

**Comment by RobinL** [comment]
Hopefully this helps explain what's going on a bit better - I think you're confusing omparisons with comparison levels in your settings 

https://moj-analytical-services.github.io/splink/topic_guides/customising_comparisons.html

**Comment by kotoroshinoto** [comment]
it is unclear how to rank multiple jaccard comparison ranges vs levenshtein ones. Wouldn't it be better to have them operate in tandem as separate comparisons?
